openwisp_radius/api/views.py

```diff
@@ -409,21 +411,15 @@ def validate_membership(self, user):
             raise serializers.ValidationError({'non_field_errors': [message]})
 
 
-class DownloadRadiusBatchPdfView(DispatchOrgMixin, APIView):
+class DownloadRadiusBatchPdfView(DispatchOrgMixin, RetrieveAPIView):
     authentication_classes = (BearerAuthentication, SessionAuthentication)
-    permission_classes = (IsAdminUser, DjangoModelPermissions)
+    permission_classes = (IsOrganizationMember, IsAdminUser, DjangoModelPermissions)
```

Comment:
didn't spot this before, I think we have to use IsOrganizationManager, because this kind of functionality is aimed at administrators, not end-users (public wifi users).

Reply:
I was under the assumption that all operators are not necessarily `is_admin` for all organizations.
I have fixed it.

Reply:
your assumption is correct, one operator will be manager of organization A but may be end-user in organization B, therefore it must not be able to access the radius batch API of organization B.